src/main/java/ladder/domain/enums/NameFormat.java

```diff
@@ -0,0 +1,30 @@
+package ladder.domain.enums;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum NameFormat {
+    DEFAULT(0, "%s "),
+    ONE(1, "  %s  "),
+    TWO(2, "  %s "),
+    THREE(3, " %s "),
+    FOUR(4, "%s "),
+    FIVE(5, "%s");
+
+    private final int nameLength;
+    private final String format;
+
+    NameFormat(int nameLength, String format) {
+        this.nameLength = nameLength;
+        this.format = format;
+    }
+
+    public static String findFormat(int nameLength) {
+        Optional<String> optionalString = Arrays.stream(values())
+                .filter(f -> f.nameLength == nameLength)
+                .map(f -> f.format)
+                .findFirst();
+
+        return optionalString.orElseGet(() -> DEFAULT.format);
```

Comment:
```suggestion
return Arrays.stream(values())
             .filter(f -> f.nameLength == nameLength)
             .map(f -> f.format)
             .findFirst()
             .orElse(DEFAULT.format);
```

좀 더 간단하게 표현할 수 있습니다.

Reply:
훨씬 더 깔끔한 코드인 것 같습니다! 반영했습니다!